README.md

```diff
@@ -88,6 +90,8 @@ It uses [node-config](https://github.com/lorenwest/node-config) to let you confi
 
 Before running the app in development, you must copy the `config/development-dist.js` into `config/development.js` (this is done by the `make install` command), and edit the server and database settings to your development environment. Same for the `test-dist.js` if you intend to run unit tests.
 
+**Note**: You need to [remove](src/admin) [all](src/api) [the](src/frontend) [demo](e2e/api) [code](e2e/frontend) before to start your project. A [pull request is in progress](https://github.com/marmelab/javascript-boilerplate/pull/22) to do so, but this will take some time to finish it.
```

Comment:
Not explicit. Instead of hyperlinks, please list the directories.

Also, s/before to start/before starting